src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/TypeThreadStaticIndexNode.cs

```diff
@@ -68,7 +68,7 @@ protected override ObjectData GetDehydratableData(NodeFactory factory, bool relo
                     // an index in the containing storage.
                     // We use a negative index to indicate that. Any negative value would work.
                     // For the purpose of natvis we will encode the offset of the type storage within the block.
-                    typeTlsIndex = - (_inlinedThreadStatics.GetTypeStorageOffset(_type) + factory.Target.PointerSize);
+                    typeTlsIndex = (-1 * (_inlinedThreadStatics.GetTypeStorageOffset(_type) + factory.Target.PointerSize));
```

Comment:
This seems strictly worse. I’d expect it to just be`-(…)` rather than multiplying by -1

(I’m not sure if we recognize and optimize that pattern either)

Reply:
Yeah... I'm ready to chuck this PR entirely. This rule already appears to have little value on the balance.